Code/GraphMol/MolDraw2D/DrawMolMCHCircleAndLine.cpp

```diff
@@ -0,0 +1,287 @@
+        make_adjusted_line(at1_cds - perp * rad, at2_cds - perp * rad, col);
+      }
+    } else {
+      auto col_rad = 2.0 * rad / hb.second.size();
+      if (drawOptions_.fillHighlights) {
+        auto p1 = at1_cds - perp * rad;
+        auto p2 = at2_cds - perp * rad;
+        std::vector<Point2D> line_pts;
+        for (size_t i = 0; i < hb.second.size(); ++i) {
+          line_pts.clear();
+          line_pts.push_back(p1);
+          line_pts.push_back(p1 + perp * col_rad);
+          line_pts.push_back(p2 + perp * col_rad);
+          line_pts.push_back(p2);
+          DrawShape *pl = new DrawShapePolyLine(
+              line_pts, lineWidth, drawOptions_.scaleBondWidth, hb.second[i],
+              true, at1_idx, at2_idx, bond->getIdx(), noDash);
+          bondHighlights.emplace_back(pl);
+          p1 += perp * col_rad;
+          p2 += perp * col_rad;
+        }
+      } else {
+        std::vector<DrawColour> cols{hb.second};
+        if (cols.size() % 2) {
+          make_adjusted_line(at1_cds, at2_cds, cols[0]);
+          cols.erase(cols.begin());
+        }
+        int step = 0;
+        for (size_t i = 0; i < cols.size(); ++i) {
+          // draw even numbers from the bottom, odd from the top
+          auto offset = perp * (rad - step * col_rad);
+          if (!(i % 2)) {
+            make_adjusted_line(at1_cds - offset, at2_cds - offset, cols[i]);
+          } else {
+            make_adjusted_line(at1_cds + offset, at2_cds + offset, cols[i]);
+            step++;
+          }
+        }
+      }
+    }
+  }
+}
+
+// ****************************************************************************
+void DrawMolMCHCircleAndLine::makeAtomHighlights(
+    std::vector<std::unique_ptr<DrawShape>> &atomHighlights) {
+  for (auto &ha : mcHighlightAtomMap_) {
+    double xradius, yradius;
+    Point2D centre;
+    int lineWidth = getHighlightBondWidth(drawOptions_, -1, nullptr);
+    calcSymbolEllipse(ha.first, centre, xradius, yradius);
+    if (ha.second.size() == 1) {
+      Point2D radii(xradius, yradius);
+      std::vector<Point2D> pts{centre, radii};
+      DrawShape *ell =
+          new DrawShapeEllipse(pts, lineWidth, true, ha.second.front(),
+                               drawOptions_.fillHighlights, ha.first);
+      atomHighlights.emplace_back(ell);
+    } else {
+      auto arc_size = 360.0 / double(ha.second.size());
+      auto arc_start = 270.0;
+      for (size_t i = 0; i < ha.second.size(); ++i) {
+        auto arc_stop = arc_start + arc_size;
+        arc_stop = arc_stop >= 360.0 ? arc_stop - 360.0 : arc_stop;
+        std::vector<Point2D> pts{centre, Point2D(xradius, yradius)};
+        DrawShape *arc = new DrawShapeArc(
+            pts, arc_start, arc_stop, lineWidth, true, ha.second[i],
+            drawOptions_.fillHighlights, ha.first);
+        atomHighlights.emplace_back(arc);
+        arc_start += arc_size;
+        arc_start = arc_start >= 360.0 ? arc_start - 360.0 : arc_start;
```

Comment:
seems clearer to just do:
```suggestion
        if(arc_start >= 360.0){
          arc_start -= 360;
        }```